source/internal/_filterMap.js

```diff
@@ -0,0 +1,10 @@
+export default function _filterMap(fn, map) {
+  var result = new Map();
+
+  for (var [key, value] of map) {
```

Comment:
We're not using `of` yet, could we re-write this to use `in` or some other equivalent?

Reply:
> `for...of` is the equivalent for `in`, and the best option. It's been widely available since 2016 and predates the 2 options of turning the iterable into an array with either `Array.from(map)` or `[...map].

I'm not arguing against `of` in general, but I don't think we should introduce it here.